src/main/java/net/glowstone/util/nbt/CompoundTag.java

```diff
@@ -124,8 +134,12 @@ public void mergeInto(CompoundTag other, boolean overwrite) {
     ////////////////////////////////////////////////////////////////////////////
     // Simple gets
 
-    public boolean getBool(String key) {
-        return containsKey(key) && getByte(key) != 0;
+    public boolean getBoolDefaultFalse(String key) {
```

Comment:
Could we just have `getBool(String)` and `getBool(String, boolean)`?

Reply:
Done; see getBoolean(String, boolean).